_data/apis.yml

```diff
@@ -101,4 +101,14 @@
   tags: [data]
   language: [it]
   slug: mic-data-sparql
+- openapi: https://raw.githubusercontent.com/APT-Servizi/openapi-ert/main/v1/api-spec.yaml
+  title: OpenData Emilia Romagna Turismo 
+  abstract: >-
+    Catalogo ufficiale degli open-data del turismo della regione Emilia-Romagna.
+  contact:
+    name: APT Servizi
+    logo: https://user-images.githubusercontent.com/1525536/134203254-f68daf19-29ad-4c7b-998e-9e61f8df4248.png
```

Comment:
Thanks for the PR @wetfire2k!

We should use a local image file instead of hotlinking it, like we did with [this one](https://github.com/italia/developers.italia.it/pull/1038/files)

Reply:
ok, i'm going to add local image

Reply:
I uploaded the file and update the link.